### DISCUSSION TITLE
Cant find directory where the downloads are saved

### DISCUSSION THREAD

**Discussion by yeeterwa** [Serene-Arc/bulk-downloader-for-reddit#471] [Q&A] [answered]
I cant find where it downloaded the images to. When I try to type something past the directory command it says unexpected argument and I cant find the default download place

**Comment by Serene-Arc** [comment, marked as answer]
There is no default download place. You have to supply it yourself. Please look at the example commands given in the README.

**Reply by yeeterwa** [reply]
Yes its good now thanks
